defs/scenarios/openstack/openstack_apache2_certificates.yaml

```diff
@@ -0,0 +1,18 @@
+requires:
```

Comment:
this will need to be a check  in itself since anything outside of checks: will be ignored.